site/app/templates/polls/AllPollsPageInstructor.twig

```diff
@@ -17,13 +17,17 @@
             </div>
         </div>
         <table id="today-table" class="table table-striped" style="width:100%;{{ dropdown_states['today'] ? '' : ' display: none;'}}">
-            <col style="width: 20%">
-            <col style="width: 20%">
-            <col style="width: 20%">
-            <col style="width: 20%">
-            <col style="width: 20%">
+	    <col style="width: 5%">
+	    <col style="width: 5%">
+	    <col style="width: 45%">
+            <col style="width: 10%">
+            <col style="width: 10%">
+            <col style="width: 10%">
+            <col style="width: 15%">
             <thead>
                 <tr>
+                    <th scope="col"></th>
```

Comment:
Just one small change, we should use `<td>` instead of `<tr>`  from [WAVE](https://wave.webaim.org/):

> #### What It Means
> A <th> (table header) contains no text.
> #### Why It Matters
> The <th> element helps associate table cells with the correct row/column headers. A <th> that contains no text may result in cells with missing or incorrect header information.
> #### How to Fix It
> If the table cell is a header, provide text within the cell that describes the column or row. If the cell is not a header or must remain empty (such as the top-left cell in a data table), make the cell a <td> rather than a <the>.

This error existed before on master, but this would be a good PR to fix the error.

Reply:
I tried to change all the relevant `<th>` tags to `<td>` like you suggested, and while it looks normal in light mode, in dark mode there are white lines only above the columns that still have the `<th>` tags. Do you think that's fine and that we should keep it that way, or should we leave it as it was before?
![image](https://user-images.githubusercontent.com/71195502/118651374-7b43c500-b7b3-11eb-87f5-b88c76efbd16.png)

Reply:
Yeah you can make them white by modifying the css class in `site/public/server.css` on ln 225 to be 
```css
.table tr, .table td[scope="col"]{
    border-top: 1px solid #ddd;
}
```
which should make the top the same color as the rest